### DISCUSSION TITLE
Welcome to 2022 NSF CSSI PI Meeting Discussions!

### DISCUSSION THREAD

**Discussion by jlmays** [cssi-pi-community/2022-meeting#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community.

**Comment by alexvakimov** [comment]
Amazing! I'm learning something new already :) Going to adopt the discussions for our projects. Thanks for organizing it and giving us an example.

**Reply by cmbarton** [reply]
> Amazing! I'm learning something new already :) Going to adopt the discussions for our projects. Thanks for organizing it and giving us an example.

@alexvakimov  Glad the conference is already having some real world benefits for the CSSI community. 

After some discussion, we decided to use the new beta Projects in GitHub for project management in our CSSI also. It seems like a very good project management tool and doesn't require that team members learn YAOP (yet another online platform).
